gui/default/index.html

```diff
@@ -344,118 +344,142 @@ <h3 class="panel-title">
+                  <div class="form-group">
+                    <div class="col-md-9">
+                      <label class="stay-logged-in">
+                        <input type="checkbox" ng-model="login.stayLoggedIn" >&nbsp;<span translate>Stay logged in</span>
+                      </label>
+                    </div>
+                    <div class="col-md-3 text-right">
+                      <button type="submit" id="submit" class="btn btn-primary" ng-disabled="login.inProgress" translate>
+                        Log In
+                      </button>
+                    </div>
+                  </div>
+
+                  <div class="row">
+                    <div class="col-md-9 login-form-messages">
+                      <p ng-if="login.errors.badLogin" class="text-danger" translate>
+                        Incorrect user name or password.
+                      </p>
+                      <p ng-if="login.errors.failed" class="text-danger" translate>
+                        Login failed, see Syncthing logs for details.
+                      </p>
+                    </div>
+                  </div>
+                </form>
+              </div>
             </div>
           </div>
-        </form>
 
-        <div class="col-md-6" style="text-align: center">
-          <p translate>
-            Or...
-          </p>
-          <div class="form-group">
-            <label>
-              <input type="checkbox" ng-model="login.stayLoggedIn" >&nbsp;<span translate>Stay logged in</span>
-            </label>
-          </div>
-          <button
-            type="button"
-            class="btn btn-primary"
-            style="margin-bottom: 0.5em;"
-            ng-click="authenticateWebauthnFinish()"
-            ng-disabled="login.inProgress || webauthn.errors.noCredentials || webauthn.errors.initFailed || locationDoesNotMatchWebauthnRpId() || isLocationInsecure()"
-            tabindex="3">
-            <span translate>Log in with WebAuthn</span>
-          </button>
+          <div class="panel panel-default">
+            <button class="btn panel-heading" data-toggle="collapse" data-parent="#login-options" data-target="#login-option-webauthn" aria-expanded="{{isDefaultLoginOption('webauthn') ? 'true' : 'false'}}">
+              <h4 class="panel-title">
+                <div class="panel-title-text" translate>WebAuthn (Passkey)</div>
+              </h4>
+            </button>
+            <div id="login-option-webauthn" class="panel-collapse collapse {{isDefaultLoginOption('webauthn') ? 'in' : ''}}">
+              <div class="panel-body">
+                <div class="form-group">
+                  <div class="col-md-6">
+                    <label class="stay-logged-in">
+                      <input type="checkbox" ng-model="login.stayLoggedIn" ng-disabled="login.inProgress || isWebauthnLoginDisabled()">
+                      &nbsp;
```

Comment:
An `&nbsp;` on a line of its own is useless.  Because the line break and indentation spaces do count as a single regular space, so wrapping before or after your non-breaking space is still done.  See the `<!-- Avoid stray space...` comments in this file for an example how to avoid that, while still wrapping lines for readability.

Reply:
Good catch, thanks. Done in 00ba31eb965e286e0f1c468cd8c04d83cbb8f3d9.